addons/binding/org.openhab.binding.meterreader/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please remove the file logo.png from the PR.

Reply:
sure

Reply:
Done!